src/Servers/HttpSys/src/AuthenticationManager.cs

```diff
@@ -63,6 +63,19 @@ public bool AllowAnonymous
     /// </summary>
     public string? AuthenticationDisplayName { get; set; }
 
+    /// <summary>
+    /// If true, the Kerberos authentication credentials are cached. Kerberos or Negotiate
+    /// authentication must be enabled. The default is false.
+    /// </summary>
+    public bool EnableKerberosCredentialCaching { get; set; }
+
+    /// <summary>
+    /// If true, the server captures the caller's credentials and uses them for Kerberos
+    /// or Negotiate authentication. Kerberos or Negotiate authentication must be enabled.
+    /// The default is false.
+    /// </summary>
+    public bool CaptureCredential { get; set; }
```

Comment:
I know we're mirroring an underlying API, but it seems like this should be "CaptureCredentials".

Reply:
`CaptureProcessCredentials`

Reply:
In 39edd844a70e4bc990fbaefd20f05f39e1cfdcd5, I went with the `CaptureCredentials` name for this option. Please let me know if it works for you.